components/profile/PinnedRepositories.js

```diff
@@ -12,7 +12,7 @@ const PinnedRepositories = ({ username, pinnedRepositories }) => {
       <p className="text-center mb-5 text-xl font-semibold">
         {"@"} {username} Pinned Repositories
       </p>
-      <div className="flex flex-wrap justify-center gap-4 max-h-[450px] overflow-y-scroll pt-2 pr-5">
+      <div className="flex flex-wrap justify-center gap-4 max-[805px]:max-h-[450px] min-h-[450px] max-[805px]:overflow-y-scroll pt-2 pr-5">
```

Comment:
Please use `max-md:` instead of `max-[805px]:`

Reply:
@Mridul2820 When I use max:md if the user has more then 3 pinned repo then it's taking large space. That's not looking good for UX. That's why i used max:[805px] which is break point for 1 card in row to 2 card in a row.
![image](https://github.com/Mridul2820/git-o-get/assets/93966956/e3f55b14-2e83-4fbc-9c63-34bc7d0b1c02)